app/models/browse_entry.rb

```diff
@@ -23,6 +24,7 @@ class BrowseEntry < ActiveRecord::Base
   accepts_nested_attributes_for :translations, allow_destroy: true
   default_scope { includes(:translations) }
 
+  before_create :set_editor_permissions
```

Comment:
yeah, that's exactly the reason I didn't put it in to the concern. And I guess the fact that we don't want it on landing pages made me consider future Classes that shouldn't have it, but let's deal with that as we need to, we can always move it out if required.